include/affordance_primitives/screw_conversions.hpp

```diff
@@ -58,7 +58,7 @@ inline Eigen::Matrix3d getSkewSymmetricMatrix(const Eigen::Vector3d& vec)
   return output;
 }
 
-inline std::string twistToStr(const geometry_msgs::msg::TwistStamped& twist)
+inline std::string twistToStr(const geometry_msgs::TwistStamped& twist)
```

Comment:
when using inline functions it's best to add the `-Winline` compiler flag so you know when your inline directives are being ignored. most probably not in this case but a good practice in larger codebases